tests/Application.UnitTests/Products/Commands/UpdateProductTests.cs

```diff
@@ -3,45 +3,39 @@
 using CleanCompanyName.DDDMicroservice.Application.Products.Commands.UpdateProduct;
 using CleanCompanyName.DDDMicroservice.Domain.Common.Exceptions;
 using CleanCompanyName.DDDMicroservice.Domain.Entities.Products;
-using FluentAssertions;
 using FluentValidation;
 using Mapster;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Logging.Abstractions;
-using Moq;
-using Xunit;
 
 namespace CleanCompanyName.DDDMicroservice.Application.UnitTests.Products.Commands;
 
 public class UpdateProductTests : ProductTestBase
 {
-    private readonly Mock<IDateTime> _dateService;
-    private readonly ILogger<UpdateProductCommandHandler> _logger;
-    private readonly Mock<IValidator<Product>> _validator;
+    private readonly Mock<IDateTime> _dateService= new();
+    private readonly Mock<ILogger<UpdateProductCommandHandler>> _logger = new();
+    private readonly Mock<IValidator<Product>> _validator= new();
```

Comment:
There is a space missing between the variable name and the '='

Reply:
Done